app/flame_chart/flame_chart.tsx

```diff
@@ -548,14 +548,14 @@ class HoveredBlockInfo extends React.Component<{ buildDuration: number }, Hovere
         <div className="hovered-block-title">{name}</div>
         <div className="hovered-block-details">
           <div>{category}</div>
-          <div>
-            <span className="data">{ts / MICROSECONDS_PER_SECOND}</span> seconds &ndash;{" "}
-            <span className="data">{(ts + dur) / MICROSECONDS_PER_SECOND}</span> seconds
-          </div>
-          <div>
+
+          <div className="duration">
             <span className="data">{dur / MICROSECONDS_PER_SECOND}</span> seconds total (
             <span className="data">{percentage}</span> of total build duration)
           </div>
+          <div>
+            @ {ts / MICROSECONDS_PER_SECOND} s &ndash; {(ts + dur) / MICROSECONDS_PER_SECOND} s
```

Comment:
can we use your nifty on decimal rounding func on these (including duration) and truncate them all at 3 decimals?

if not, totally fine

Reply:
Good idea, that'll help reduce information overload. Done!